features/promotion/receiving_percentage_discount_on_items_on_price_range.feature

```diff
@@ -0,0 +1,65 @@
+@promotion
+Feature: Receiving percentage discount on products from specific price range
+    In order to pay less while buying goods from specific price range
+    As a Customer
+    I want to receive discount for my purchase
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "€100.00"
+        And the store has a product "PHP Mug" priced at "€20.00"
+        And there is a promotion "Christmas promotion"
```

Comment:
Maybe `"Christmas sale"` would be more appropriate, because the `promotion "X promotion"` IMO sounds silly.